docs/source/developers/guide/step_by_step/building.rst

```diff
@@ -143,6 +143,16 @@ Building other Arrow libraries
 
          - :ref:`build_pyarrow_win`
 
+   .. tab:: Building the R package
```

Comment:
@thisisnic in https://github.com/apache/arrow/pull/12119, you moved the (formerly empty) tab down the file, and I now moved it back. 
The reason is that the two `.. tab::` items need to be together under `.. tabs::` (the current doc build was broken because of this)

Reply:
We should, but without the heavy builds, like java.

Reply:
Something like https://github.com/apache/arrow/pull/12171

Reply:
Oops, sorry, and thanks for flagging this up @jorisvandenbossche !